docs/manage-metadata-using-gravitino.md

```diff
@@ -733,6 +733,159 @@ In addition to the basic settings, Gravitino supports the following features:
+| `list`            | Partition the table by a list value                          | Any                                                                                 | Any         | `{"strategy":"list","fieldNames":[["dt"],["city"]]}`             | `Transforms.list(new String[] {"dt", "city"})`  | `PARTITION BY list(dt, city)`      |
+| `range`           | Partition the table by a range value                         | Any                                                                                 | Any         | `{"strategy":"range","fieldName":["dt"]}`                        | `Transforms.range(20, "score")`                 | `PARTITION BY range(score)`        |
+
+Except the strategies above, you can use other functions strategies to partition the table, for example, the strategy can be `{"strategy":"functionName","fieldName":["score"]}`. The `functionName` can be any function name that you can use in SQL, for example, `{"strategy":"toDate","fieldName":["score"]}` is equivalent to `PARTITION BY toDate(score)` in SQL.
+For complex function, please refer to `FunctionPartitioningDTO`.
+
+The following is an example of creating a partitioned table:
+
+<Tabs>
+<TabItem value="Json" label="Json">
+
+```json
+[
+  {
+    "strategy": "identity",
+    "fieldName": [
+      "score"
+    ]
+  }
+]
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+new Transform[] {
+    // Partition by score
+    Transforms.identity("score")
+    }
+```
+
+</TabItem>
+</Tabs>
+
+
+#### Bucketed table
+
+- Strategy. It defines in which way you bucket the table.
+
+| Bucket strategy | Description                                                                                                          | Source types | Result type | Json    | Java             |
+|-----------------|----------------------------------------------------------------------------------------------------------------------|--------------|-------------|---------|------------------|
+| hash            | Bucket table using hash. The data will be distributed into buckets based on the hash value of the key.               | Any          | int         | `hash`  | `Strategy.HASH`  |
+| range           | Bucket table using range. The data will be divided into buckets based on a specified range or interval of values.    | Any          | Source type | `range` | `Strategy.RANGE` |
+| even            | Bucket table using even. The data will be evenly distributed into buckets, ensuring an equal distribution of data.   | Any          | Source type | `even`  | `Strategy.EVEN` |
+
+- Number. It defines how many buckets you use to bucket the table.
+- Function arguments. It defines which field or function should be used to bucket the table. Gravitino supports the following three kinds of arguments, for more, you can refer to Java class `FunctionArg` and `DistributionDTO` to use more complex function arguments.
```

Comment:
It defines the arguments of the above strategy

Reply:
Okay